.github/workflows/ci.yml

```
@@ -101,6 +101,23 @@ jobs:
       - name: Machete
         uses: bnjbvr/cargo-machete@main
 
+  # Checks for duplicate version of package
+  cargo-vendor:
+    name: Cargo Vendor
+    runs-on: 8core_ubuntu_latest_runner
+    steps:
+      - uses: actions/checkout@v4
+      - uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          cache: ${{ github.ref == 'refs/heads/main' }}
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ". -> target/pixi"
+          key: ${{ hashFiles('pixi.lock') }}
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+      - name: Test cargo vendor
+        run: cargo vendor --versioned-dirs
```

```suggestion
        run: cargo vendor --versioned-dirs --locked
```
If I understand this correctly, we should use the `--locked` flag.